mParticle-Firebase-Analytics/MPKitFirebaseAnalytics.h

```diff
@@ -0,0 +1,37 @@
+//
+//  MPKitFirebaseAnalytics.h
+//
+//  Copyright 2018 mParticle, Inc.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+#import <Foundation/Foundation.h>
+#if defined(__has_include) && __has_include(<mParticle_Apple_SDK/mParticle.h>)
+#import <mParticle_Apple_SDK/mParticle.h>
+#else
+#import "mParticle.h"
+#endif
+
+static NSString *const googleAppIDKey = @"firebaseAppId";
```

Comment:
The field/variable is actually call google App ID in the googleInfo.plist, firebase console, and in FIROptions method below. Android and google internally just use this. So I figured keeping this would help make it more obvious that they represent the same thing.